package.json

```diff
@@ -44,18 +46,14 @@
     "ember-prop-types": "3.2.0",
     "ember-resolver": "^2.0.3",
     "ember-sinon": "0.6.0",
-    "ember-test-utils": "1.2.2",
     "eslint": "^3.8.1",
     "eslint-config-frost-standard": "^4.0.0",
-    "loader.js": "^4.0.1"
+    "loader.js": "^4.0.10"
   },
-  "keywords": [
-    "ember-addon"
-  ],
-  "dependencies": {
-    "ember-cli-babel": "^5.1.6"
+  "engines": {
+    "node": ">= 0.12.0"
```

Comment:
This aligns with the current recommendation by the ember-cli team. IIRC 0.12 won't be supported with the next release of Ember CLI, so that's good news.